tests/js/spec/utils/dashboards/issueFieldRenderer.spec.tsx

```diff
@@ -9,14 +9,18 @@ import {
 import GroupStore from 'sentry/stores/groupStore';
 import MemberListStore from 'sentry/stores/memberListStore';
 import ProjectsStore from 'sentry/stores/projectsStore';
+import {User} from 'sentry/types';
```

Comment:
since we are working on this file, can we update the test `it('can render assignee'` to have only one await? 🙏

https://github.com/getsentry/sentry/blob/a5f32835bdbecb7852c83f14fb0f4afe0e58b7fc/tests/js/spec/utils/dashboards/issueFieldRenderer.spec.tsx#L114

```
    expect(screen.getByText('TU')).toBeInTheDocument();
      userEvent.hover(screen.getByText('TU'));
      expect(await screen.findByText('Assigned to')).toBeInTheDocument();
      expect(screen.getByText('Test User')).toBeInTheDocument();
      expect(screen.getByText('Based on')).toBeInTheDocument();
      expect(screen.getByText('commit data')).toBeInTheDocument();
```

Reply:
Yup, updated to just one await!